Friendly-plans/app/build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
 
 android {
     compileSdkVersion 24
-    buildToolsVersion '24.0.2'
+    buildToolsVersion '25.0.0'
```

Comment:
You're build is failing because travis cannot cover the dependencies. I don't think, travis supports higher versions at the moment. Move the sdk, gradle and wrapper versions to previous one. It should help or try with updating travis.yml configuration file.

Reply:
I've moved versions to previous one, but now travis returns this error: "The SDK Build Tools revision (24.0.2) is too low for project ':app'. Minimum required is 25.0.0".  :(